ccd-definition/FixedLists/GeneralApplicationTypesGAspec-GAR2.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "ID": "GeneralApplicationTypesGAspec",
+    "ListElementCode": "SET_ASIDE_JUDGEMENT",
```

Comment:
I would suggest to review the Display order as if both files are imported there will be conflict in displayorder

Reply:
I guess we can ignore the -GAR2.json file in lower env too. @vasudevganesanhmcts your thoughts?